package-examples/ghost/README.md

```diff
@@ -0,0 +1,32 @@
+### Ghost Application
+
+"Ghost is a powerful app for new-media creators to publish, share, and grow a business around their content. It comes with modern tools to build a website, publish content, send newsletters & offer paid subscriptions to members."
+https://ghost.org/
+
+### Quick start
+
+#### Get KPT Pacakge
+```bash
+export NAMESPACE=<YOUR NAMESPACE>
+# make sure the namespace is correct and exists. Otherwise, create the namespace
+kubectl create namespace ${NAMESPACE}
```

Comment:
why can't we add namespace to the package itself ? 

If you used helm chart to get the manifests, you can use `--create-namespace` arg to add it.

That will make the workflow truly declarative.

Reply:
The ability to create a namespace would imply that the app team had cluster admin privilege. It also would bring in the need for the flexibility of the namespace provisioning scenario we've been working on: resource quota, limit range, network policy, etc.

Reply:
We should have an example namespace blueprint for this, which could be used instead of kubectl create namespace.

Reply:
> The ability to create a namespace would imply that the app team had cluster admin privilege. It also would bring in the need for the flexibility of the namespace provisioning scenario we've been working on: resource quota, limit range, network policy, etc.

Aah.. completely missed that. Good pt.